File: Lab/OddVsEven.c
#include <stdio.h>
void main ()
{
    int odd,even;
    int i=0;
    int u_input;

    for (i=0; i<8; i++)
    {
        scanf ("%d",&u_input);
        if ((u_input%2)==0)
        {
            even += u_input;
        }
        else if (u_input==0) {}
        else
        {
            odd += u_input;
        }
    }
    if (odd==even)
    {
        printf ("equal");
    }
    else if (odd>even)
    {
        printf ("odd");
    }
    else
    {
        printf ("even");
    }
    printf ("\n%d\n%d",even,odd);
}

File: ComproII/lib/Allcode/Datatype_Array1D/odd_vs_even.java
import java.util.Scanner;

public class odd_vs_even {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int total_even=0;
        int total_odd=0;
        for (int i = 0; i<8; i++)
        {
            int x;
            x = scan.nextInt();
            if (x%2 == 0)
            {
                total_even += x;
            }
            else
            {
                total_odd += x;
            }
        }

        if (total_even > total_odd)
        {
            System.out.println("even");
            System.out.println(total_even);
            System.out.println(total_odd);
        }
        else if (total_even == total_odd)
        {
            System.out.println("equal");
            System.out.println(total_even);
            System.out.println(total_odd);
        }
        else
        {
            System.out.println("odd");
            System.out.println(total_even);
            System.out.println(total_odd);
        }
    }
}
